### PR TITLE
hunspell-el: include homepage

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- Inclusion of "homepage" to package.yml
- Part of https://github.com/getsolus/packages/issues/411

**Test plan**
- Checked that homepage was added correctly

**Check list**
- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/h/hunspell-el/package.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name       : hunspell-el
-version    : 0.9
-release    : 4
+version    : 0.9.0
```
Comment (algent-al) [inline packages/h/hunspell-el/package.yml:2]: No need for extra `.0`.  Corrected should be `version    : '0.9'`

**Review by algent-al — CHANGES_REQUESTED** [review]

**Comment by androidnisse** [comment]
@algent-al should be fixed now.

**Review by algent-al — APPROVED** [review]
LGTM, thanks!

**Review by algent-al — APPROVED** [review]
LGTM, thanks!

**Comment by androidnisse** [comment]
@algent-al could this be merged now?

**Pull request merged**